src/group/groupFragments.js

```diff
@@ -57,15 +57,15 @@ export const groupsListFields = `
 
 export const groupMembersPending = `
   fragment groupMembersPending on GroupNode {
-    pending: memberships(membershipStatus: "pending") {
+    pending: memberships(membershipStatus: PENDING) {
```

Comment:
question: This changes are due to [better enum support](https://github.com/graphql-python/graphene/wiki/v3-release-notes#better-enum-support), right? Checking I understand correctly.